videos/tasks.py

```diff
@@ -133,7 +136,31 @@ def delete_s3_objects(
 @app.task(acks_late=True)
 def update_transcripts_for_video(video_id: int):
     """Update transcripts for a video"""
-    threeplay_api.update_transcripts_for_video(Video.objects.get(id=video_id))
+    video = Video.objects.get(id=video_id)
+    if threeplay_api.update_transcripts_for_video(video):
+        website = video.website
+        if is_ocw_site(website):
+            search_fields = {}
+            search_fields[
+                "metadata__" + settings.FIELD_FILETYPE.replace(".", "__")
+            ] = "Video"
+            search_fields[
+                "metadata__" + settings.YT_FIELD_ID.replace(".", "__")
+            ] = video.youtube_id()
+
+            for video_resource in website.websitecontent_set.filter(**search_fields):
+                metadata = video_resource.metadata
+                set_dict_field(
+                    metadata,
+                    settings.YT_FIELD_TRANSCRIPT,
+                    video.pdf_transcript_file.name,
```

Comment:
`name` returns something like `courses/ocw-course-site-2/13ek-M7C5MGV7Ge5FB7JNCVtk-4Nij8Ik_transcript.pdf` but `url` would return `https://<bucket_name>.s3.amazonaws.com/courses/ocw-course-site-2/13ek-M7C5MGV7Ge5FB7JNCVtk-4Nij8Ik_transcript.pdf`

Wondering if `url` should be used instead, to make it consistent with what's written out to github for image files, for example: https://github.com/mbtestorg1/ocw-course-site-2/blob/main/content/resources/saguaro-np.md

Though maybe that image file value is going to change too.  Might be worth checking with George or Carey about that.